src/redux/wallet/selectors.js

```diff
@@ -19,3 +32,181 @@ export const getCurrentWalletBalance = (symbol) => createSelector(
   [ getCurrentWallet ],
   (currentWallet) => currentWallet.balances().item(symbol)
 )
+
+export const selectMainWalletBalancesListStore = (state) =>
+  state.get(DUCK_MAIN_WALLET).balances().list() // BalancesCollection, array of BalanceModel
+
+export const selectMainWalletTokensStore = (state) =>
+  state.get(DUCK_TOKENS) // TokensCollection, array of TokenModel
+
+export const selectMainWalletAddressesListStore = (state) =>
+  state.get(DUCK_MAIN_WALLET).addresses().list() // This is an instance of MainWalletModel
+
+export const selectMarketPricesListStore = (state) => state.get(DUCK_MARKET).prices
+export const selectMarketPricesSelectedCurrencyStore = (state) => state.get(DUCK_MARKET).selectedCurrency
+
+/**
+ * WALLET SECTIONS
+ * 
+ *  Usage example:
+ * 
+ * 
+ * const getSectionedWallets = makeGetSectionedWallets()
```

Comment:
Do we need so detailed usage example?

Reply:
I suppose yes, we do. It may take a time to read and understand reselect's documentation.
Also it's useful for copy-paste for now.